api/src/model/request/play_asset_pack_remove_request.gd

```diff
@@ -22,6 +22,7 @@
 # the updated states from this object once the request completes.
 #
 # ##############################################################################
+# warning-ignore:unused_signal
```

Comment:
Since we are using ```	call_deferred("emit_signal", "request_completed", true, null)``` here, Godot runtime would suppose that this signal is unused since there is no direct emit_signal calls.

Reply:
Updated this comment.